src/scripts/migrate.ts

```diff
@@ -6,8 +6,7 @@ const logger = root_logger.child({ source: 'migrate' });
 logger.info('Starting migrations...');
 migrate()
   .then(() => {
-    logger.info('Migrations complete.');
-    process.exit();
+    process.stdout.write('Migrations complete.\n');
```

Comment:
Should it be `logger.info`?

Reply:
Oh yes -- I kept the wrong line when resolving the merge conflict